Mathlib/GroupTheory/GroupAction/Hom.lean

```diff
@@ -947,3 +948,9 @@ def inverse {S₁ : Type*} [Semiring S₁] [MulSemiringAction M S₁]
 end MulSemiringActionHom
 
 end DistribMulAction
+
+lemma IsSMulRegular.of_injective {R M : Type*} [SMul R M]
```

Comment:
Current convention would rather be
```suggestion
lemma Function.Injective.isSMulRegular {R M : Type*} [SMul R M]
```
although I agree we might want to review that...

Reply:
No problem then